test/tools/d_do_test.d

```diff
@@ -895,11 +897,11 @@ int runBashTest(string input_dir, string test_name)
     version(Windows)
     {
         auto process = spawnShell(format("bash %s %s %s",
-            buildPath("tools", "sh_do_test.sh"), input_dir, test_name));
+            buildPath(scriptDir, "tools", "sh_do_test.sh"), input_dir, test_name));
     }
     else
     {
-        auto process = spawnProcess(["./tools/sh_do_test.sh", input_dir, test_name]);
+        auto process = spawnProcess([scriptDir ~ "/tools/sh_do_test.sh", input_dir, test_name]);
```

Comment:
The `/` will probably work fine in the posix/Bash environment.  I don't know if it will or won't cause problems on Windows if we ever get to a point where we can run this outside of that environment.  I suggest using `std.path.pathSeparator`.

Reply:
I went with `buildPath`